.dockerignore

```diff
@@ -2,8 +2,13 @@
 *
 
 # Make exceptions for what's needed
-!snekbox
+!.git/
```

Comment:
This could be further filtered to speed up transfer time, but I doubt it is worth spending time on.